qiskit_machine_learning/neural_networks/two_layer_qnn.py

```diff
@@ -18,17 +18,25 @@
 from qiskit import QuantumCircuit
 from qiskit.opflow import PauliSumOp, StateFn, OperatorBase, ExpectationBase
 from qiskit.providers import Backend
-from qiskit.utils import QuantumInstance
+from qiskit.utils import QuantumInstance, deprecate_function
 
 from .opflow_qnn import OpflowQNN
 from ..utils import derive_num_qubits_feature_map_ansatz
 
 
 class TwoLayerQNN(OpflowQNN):
-    """Two Layer Quantum Neural Network consisting of a feature map, a ansatz,
+    """Pending deprecation: Two Layer Quantum Neural Network consisting of a feature map, a ansatz,
     and an observable.
     """
 
+    @deprecate_function(
```

Comment:
Jus checking - If I create a TwoLayerQNN do I just get the one deprecation message or a second when it calls super.init0 from OpflowQNN. I am thinking the latter is more an internal call so I wold just have the one message.

Reply:
By default, there are no warnings printed. If I enable them, I get two warning. I don't know if that should be changed. I'd leave as now.

Reply:
Possibly the TwoLayerQNN could suppress the warning around its call to super().init We may want to check that when we change to deprecated as then the warnings will print by default.